index.js

```diff
@@ -97,7 +97,7 @@ SassCompiler.prototype._nativeCompile = function(source, callback) {
       callback(null, data);
     }),
     error: (function(error) {
-      callback(error);
+      callback(error.message);
```

Comment:
What I meant was `error.message || util.inspect(error)` in case of future unexpected changes so it won't break as badly as it did this time